gpu_bdb/bdb_tools/q05_utils.py

```diff
@@ -94,11 +97,16 @@ def build_and_predict_model(ml_input_df):
     results_dict = {}
     y_pred = model.predict(X)
 
-    results_dict["auc"] = roc_auc_score(y.values_host, y_pred.values_host)
```

Comment:
We use `cuML` models for both CPU and GPU, we should use `sklearn` for CPU and `cuML` for GPU.